src/views/day/loop.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Schedule Day View Loop
+ * This file sets up the structure for the Schedule Day View loop.
+ *
+ * While this plugin is active, you cannot override this template in your own theme because it's loaded via a filter from tribe_get_template_part() in this plugin's index.php.
+ *
+ * @version 1.0.0
+ * @package TribeEventsCalendar
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+global $post, $wp_query;
+
+$prefix = Tribe__Extension__Schedule_Day_View::PREFIX;
+
+$current_time_slot = null;
+$php_class         = Tribe__Extension__Schedule_Day_View::instance();
+$today             = $php_class->today();
+$container_class   = $php_class->array_of_container_classes_based_on_today_to_string();
+
+$all_day_text = Tribe__Extension__Schedule_Day_View::instance()->get_all_day_text();
+
+$now = Tribe__Extension__Schedule_Day_View::instance()->now_timestamp();
+?>
+
+<div
+	id="tribe-events-day"
+	class="tribe-events-loop<?php echo esc_attr( $container_class ); ?>"
+	data-tribe-timezone="<?php echo esc_attr( Tribe__Events__Timezones::wp_timezone_string() ); ?>"
+	data-tribe-now="<?php echo esc_attr( $now ); ?>"
+>
+
+	<?php
+	// Used by Tribe__Extension__Schedule_Day_View::setup_loop()
+	do_action( $prefix . '_inside_before_loop' );
```

Comment:
No need to change this, but it seems like it would cause devs extra work to discover the full hook name (they need to find where $prefix is defined and then they need to find the class constant definition) ... on that level I wonder if we're making things difficult for the very folks we hope might use these hooks, for our own comfort.

Reply:
Valid concern that I hoped wouldn't be an issue (sounds like it's not) because I personally think it's the best (DRYest) way and only minimally more complex. I'll choose to leave it as-is.